PYME/experimental/isosurface.py

```diff
@@ -25,4 +26,71 @@ def isosurface(data, isolevel, voxel_size=None, origin=None, remesh=False):
     if remesh:
         T.remesh()
     
-    return T
\ No newline at end of file
+    return T
+
+def clamp(v, lo, hi):
```

Comment:
This also has a numpy equivalent

Reply:
Ah, `np.clip`. I didn't know this existed.